src/test/game/CommandlineUiTest.java

```diff
@@ -9,22 +10,40 @@
 import java.io.PrintStream;
 
 import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
 
 public class CommandlineUiTest {
 
+    private ByteArrayOutputStream output;
+    private InputStream input;
+    private CommandlineUi ui;
+
+    @Before
+    public void setUp() {
+        output = new ByteArrayOutputStream();
+        input = new ByteArrayInputStream("".getBytes());
+        ui = new CommandlineUi(new PrintStream(output), input);
```

Comment:
Instead of this code living in the `setUp` method, you could have it in a private helper function that can take in the arguments that tend to change with each test. That would help eliminate duplicated boilerplate code like the one in `getMoveFromUser` test.